src/styles/catalog.scss

```diff
@@ -0,0 +1,8 @@
+.catalog {
+  padding-top: $catalog-padding-top;
+  padding-bottom: $catalog-padding-bottom;
+  max-width: calc((#{$card-width} * 4) + (#{$card-margin-right} * 3));
```

Comment:
one of the tasks:
there should always be 4 cards in a row (not 2, 3 or 5)

if you use width, instead of max-width, you will achieve this result

Reply:
your tests did not pass, because you set width to main, it would be better to create some container in main

Reply:
ty you for this comment;)